### ISSUE TITLE
What To Contribute? 

### ISSUE THREAD

**Issue by deadline007** [Rishikesh-12/HacktoberFest2020#622]
You can ask any doubts reagarding what to contribute .
Just Tag @Rishikesh-12

**Comment by saubhagya-gupta** [comment]
Hi @Rishikesh-12 , I have added #631 PR which is for N- Queens problem in Java. Will it be a valid PR to be approved by you ?
Please review and let me know.

**Comment by rishavjaiss** [comment]
Hey, how to contribute? @Rishikesh-12

**Comment by nehal6657** [comment]
how to contribute

**Comment by mayankparida** [comment]
Hey I would like to add a blackjack casino game made using python. Could you please allow me to make the respective contribution?

**Comment by Rishikesh-12** [comment]
> Hey, how to contribute? @Rishikesh-12

Hello @rishavjaiss you can read the `README.md` file

**Comment by Rishikesh-12** [comment]
> how to contribute

Hello @nehal6657  you can read the `README.md` file

**Comment by Rishikesh-12** [comment]
> Hey I would like to add a blackjack casino game made using python. Could you please allow me to make the respective contribution?

@mayankparida We would love to merge the pull request if it passes the **review** .

**Comment by mayankparida** [comment]
I had sent the pull request yesterday please do check n merge. Thank you!

**Comment by karthick191999** [comment]
hey can u merge my pull request @Rishikesh-12  #564  it has passed the review

**Comment by Rishikesh-12** [comment]
Hello @karthick191999 we will surely merge the pull request if it passes the review.

**Comment by bighnesh0404** [comment]
#684 please the merge the pull request

**Comment by rishavjaiss** [comment]
Hi @Rishikesh-12 , Please merge #675

**Comment by Rishikesh-12** [comment]
Hello @rishavjaiss and @bighnesh0404 Please refrain from spamming, we will merge every pull request if it gets accepted after review from the reviewers.

**Comment by yashjhaveri05** [comment]
Hey @Rishikesh-12, I would like to add the functionality to subtract two polynomials in the C folder for polynomials!!So could I contribute by doing that?

**Comment by deadline007** [comment]
Hello @yashjhaveri05 , We would love to see you contribute with that.

**Comment by priyajames29** [comment]
Hi, I would like to contribute a simple translator program in python.  Can I make the contribution?

**Comment by yashjhaveri05** [comment]
Hello!!Thankyou very much with the opportunity!!Ill send over the code by tomorrow!!From: Ayush ShuklaSent: 10 October 2020 23:13To: Rishikesh-12/HacktoberFest2020Cc: Yash Jhaveri; MentionSubject: Re: [Rishikesh-12/HacktoberFest2020] What To Contribute? (#622) Hello @yashjhaveri05 , We would love to see you contribute with that.—You are receiving this because you were mentioned.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by yashjhaveri05** [comment]
> Hello @yashjhaveri05 , We would love to see you contribute with that.
Hey @Rishikesh-12,I have made the pull request, please do review it!!

**Comment by Rishikesh-12** [comment]
> Hi, I would like to contribute a simple translator program in python. Can I make the contribution?

Yes @priyajames29 . We would love to see you contribute with that.

**Comment by zahinekbal** [comment]
@Rishikesh-12 hey I wanna contribute 
Can I add algorithms of datastructure ?

**Comment by atulnath977** [comment]
Hi, sent a PR for a PigGame created in JS. 
Please do review and merge.

**Comment by axayjha** [comment]
Hey @Rishikesh-12 , I have made a PR. Please review

**Comment by Rishikesh-12** [comment]
> @Rishikesh-12 hey I wanna contribute
> Can I add algorithms of datastructure ?

Hello @zahinekbal , We would love to see you contributing with that ,

**Comment by Rishikesh-12** [comment]
> Hey @Rishikesh-12 , I have made a PR. Please review
> #833

Hello @axayjha  Please refrain from spamming, we will merge every pull request if it gets accepted after review from the reviewers.

Thank you for contributing

**Comment by Aarti002** [comment]
hello @Rishikesh-12 ,i have added a html file it will be very helpful if u review my page and also i had already made one another PR it was a code in python i guess 5 to 7 days before but still no one has reviewed it! Plz review my work!!!
